README.md

```diff
@@ -41,6 +41,28 @@ Utilize Plugins to get up and running quickly. Here's a quick list of Stanza's m
 
  These are many of the Plugins supported by Stanza, with more being developed all the time. View a full list of Plugins [here](https://github.com/observIQ/stanza-plugins/tree/master/plugins).
 
+## Supported Operating Systems
+
+### Windows
+- 2012
+- 2012 r2
+- 2016
+- 2019
+- 2022
+
+### Linux
+- Ubuntu 16.04, 18.04, 20.04
+- Debian 9, 10, 11
+- Alma, Rocky 8
+- Red Hat 7, 8
```

Comment:
I think we can call out centos 7 and centos 8 stream

Reply:
added those in